### PR TITLE
[new] add checklist-model@0.6.0 with git auto-update

### LINKED ISSUE #6136: [Request] Add checklist-model
**Library name:** checklist-model
**Git repository url:** https://github.com/vitalets/checklist-model
**npm package url(optional):** https://www.npmjs.com/package/checklist-model
**License(s):** MIT
**Official homepage:** http://vitalets.github.io/checklist-model/

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Amomo**
Hi @LeaYeh 

For #6136, I add checklist-model at v0.6.0 and add git auto-update config in its package.json.
repo: https://github.com/vitalets/checklist-model

This library don't provide keywords, so I pick some word from README.md.

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `33`, Star `655`, Fork `142`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by LeaYeh** [comment]
i think this pr is ok.
@PeterDaveHello

File: ajax/libs/checklist-model/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "license": "MIT",
+  "keywords": [
+    "checklist",
+    "checkboxes",
+    "AngularJS"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://vitalets/checklist-model.git",
```
Comment (PeterDaveHello) [inline ajax/libs/checklist-model/package.json]: What's this?

Comment (LeaYeh) [inline ajax/libs/checklist-model/package.json]: git://github.com/vitalets/checklist-model.git

Comment (PeterDaveHello) [inline ajax/libs/checklist-model/package.json]: So is this really okay? I don't think so.

**Comment by Amomo** [comment]
@LeaYeh I have rebased it, could you come to check this PR? Thank you~

**Comment by Amomo** [comment]
@Piicksarn could you help me check this PR? Thank you~ :-D

**Comment by Piicksarn** [comment]
@Amomo, sorry for my delay!
Would u mind to rebase it again?

**Comment by Amomo** [comment]
@Piicksarn I have rebased my PR to latest branch, could you check it again?
Thank you~ :-D

**Pull request merged**